### ISSUE TITLE
XOnline Dash only working for some?

### ISSUE THREAD

**Issue by Voxel9** [Cxbx-Reloaded/Cxbx-Reloaded#493]
So as #492 shows, XOnline Dash/Xbox Live Dash now appears to run on the latest version of Cxbx-r. However, I still seem to be running into exceptions with it, even when I have cleared the entire HLE cache and tried every settings combination possible:

![image](https://user-images.githubusercontent.com/16278868/27631832-caac5598-5bf1-11e7-8333-d4d6cb71dfcd.png)
![image](https://user-images.githubusercontent.com/16278868/27631847-d8aaee16-5bf1-11e7-9f7a-0d38dcc26cc1.png)

Maybe there is a specific XOnline Dash version I need to use? Currently I am only testing with the XOnline Dash from the 4920 Dashboard.

As usual, I'm running on an AMD CPU/GPU, on Windows 10.
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/files/1108285/KrnlDebug.txt)

**Comment by LukeUsher** [comment]
Interesting, thanks for letting me know I'll investigate this.

The first error is normal: XOnline doesn't like it that we apply patches, it validates the Xbe contents amd throws an error, ignoring that usually makes it launch just fine.

No idea what's going on with the unimplemented opcode, I'll have a look tonight and get back to you.

**Comment by LukeUsher** [comment]
Update: It looks like this is in-fact a different version of XOnlineDash than the one I was testing with, this issue is most likely something as simple as a missing HLE/OOVPA entry

**Comment by Voxel9** [comment]
`EDIT: A quick thought, did you install the dashboard? The whole dashboard needs to be located at %AppData%\Cxbx-Reloaded\EmuDisk\Partition2\

So in that folder should be your xboxdash.xbe + all it's files and subfolders`
Yeah, I properly installed the dash to the appropriate folder in appdata; known this for many months now :p 

But anyways, thanks for the replies, much appreciated. :) Any idea which version you used, btw?

**Comment by LukeUsher** [comment]
I also used 4920 dashboard, but it would appear our XOnlineDash versions differ. (The HLE cache hashes are not the same) can you upload an Xbe dump? (Edit => Dump Xbe Info to => File)

For comparison purposes, I have attached mine
[Xbe.txt](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/files/1109629/Xbe.txt)

**Comment by Voxel9** [comment]
Well I'll be damned! I redownloaded the entire 4920 dashboard files and now XOnline Dash works! Maybe I did have a different version after all, perhaps. I'm an idiot sometimes 😖 

![image](https://user-images.githubusercontent.com/16278868/27653555-5502d164-5c37-11e7-89ab-cb8d9394c514.png)

Unfortunately I deleted the XOnlineDash that I was testing, so I never got a chance to dump the Xbe. But on the plus side, now my Xbe dump matches your dump after replacing the different version.

Ever so sorry to waste your time!
